### DISCUSSION TITLE
这个 redis db 的 pr 官方啥时候合呀

### DISCUSSION THREAD

**Discussion by SaneHe** [zeromicro/go-zero#1647] [Q&A] [answered]
[redis db select](https://github.com/zeromicro/go-zero/pull/1601)  具体时间有吗  官方可以把相关 roadmap 发出来吗 或者有地址在哪里呀

**Comment by kevwan** [comment, marked as answer]
这个PR没有合并计划呢

我不支持redis使用多DB

**Reply by SaneHe** [reply]
ok
